docker/ml-testing-toolkit/spec_files/user_config.json

```diff
@@ -29,9 +29,9 @@
   "SEND_CALLBACK_ENABLE": true,
   "FSPID": "switch",
   "DEFAULT_USER_FSPID": "userdfsp",
-  "TRANSFERS_VALIDATION_WITH_PREVIOUS_QUOTES": true,
-  "TRANSFERS_VALIDATION_ILP_PACKET": true,
-  "TRANSFERS_VALIDATION_CONDITION": true,
+  "TRANSFERS_VALIDATION_WITH_PREVIOUS_QUOTES": false,
```

Comment:
done. I've added a comment in sdk-scheme-adapter/src/test/integration/lib/Outbound/simpleTransfers.test.js explaining why these values are false since I can't comment in the json file.